src/SFA.DAS.RoATPService.Application/Validators/ProviderTypeValidator.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.RoATPService.Application.Validators
+{
+    public class ProviderTypeValidator : IProviderTypeValidator
+    {
+        public bool IsValidProviderTypeId(int providerTypeId)
+        {
+            return (providerTypeId >= 1 && providerTypeId <= 3);
```

Comment:
Does this exist in a lookup table? If not, fine, but if so, that should be source of validation.  If that's a faff, then raise a tech debt story to do this in a future story.

Reply:
This is in a database lookup table, however the number of provider types should be static, as it would require changes to the apply journey if a new one was added